### ISSUE TITLE
Any way to manually add support for a site?

### ISSUE THREAD

**Issue by hacktek** [skorokithakis/netproxy#3]
I'd like to add support for www.slacker.com. I understand that this is based on haproxy but I guess that there isn't a general match to resolve the SNI hostname or the host header and there are specific rules for the hostnames provided in the hosts file, is this correct?

If I use a non-supported hostname, I get this:

$ curl -v -k -D - -so /dev/null -H "Host: www.slacker.com" https://x.x.x.x/
* Hostname was NOT found in DNS cache
*   Trying x.x.x.x...
* Connected to x.x.x.x (x.x.x.x) port 443 (#0)
* Server aborted the SSL handshake
* Closing connection 0

$ curl -D - -so /dev/null -H "Host: www.slacker.com" http://x.x.x.x/
HTTP/1.0 503 Service Unavailable
Cache-Control: no-cache
Connection: close
Content-Type: text/html

I can edit the haproxy config to account for this hostname if necessary but since this is dockerized and due to my limited experience with this I ask: can I edit configs inside of the docker or does that need a new release of the container?

Thanks!

**Comment by skorokithakis** [comment]
All you need to do to add a site is clone trick77/tunlr-style-dns-unblocking yourself and add your servers to the `config.json`. You may need to take the clone line out of the Dockerfile, since you'll have already done it, but that's it, the Dockerfile will run and generate the files from your modified `config.json`.
